lib/ansible/modules/cloud/ovirt/ovirt_storage_domain.py

```diff
@@ -61,7 +61,7 @@
         description:
             - "Function of the storage domain."
             - "This parameter isn't idempotent, it's not possible to change domain function of storage domain."
-        choices: ['data', 'iso', 'export']
+        choices: ['data', 'iso', 'export', 'managed_block_storage']
```

Comment:
You can remove this.

Reply:
When i leave it empty it sets domain_function to Data but it should be managed_block_storage so i did workaround to check the storage type